### PR TITLE
Change arrow animation to compass animation on Main page

### PR DESCRIPTION
**Pull request by NadiiaLashtun**

### CODE REVIEW

File: src/components/MainSection.jsx [hunk 1]
```diff
@@ -1,36 +1,45 @@
...
                 Color Compass
               </h1>
-              <p className="w-2/3 md:w-[460px]">
+              <p className='w-2/3 md:w-[460px]'>
                 Lorem ipsum dolor sit amet, consectetur adipiscing elit.
                 Phasellus mattis tortor ut tortor cursus, ac maximus tortor.
               </p>
+              <a
+                href='#'
+                className='animate-pulse hover:[animation-play-state:paused]'
```
Comment (mnichols08) [inline src/components/MainSection.jsx:24]: I love the animation still and I really enjoy the position on mobile.
Haven't actually looked on desktop yet.

I wonder could we use this to create a loading spinner too?

File: src/img/compass-circle.svg
Comment (mnichols08) [inline src/img/compass-circle.svg:1]: Separating these images is so freaking cool and I love that you made the center animate not the entire image.

Just an incredible touch - thank you!

**Review by mnichols08 — APPROVED** [review]
Keep up the hard work. I really enjoy seeing your attention to details.

**Review by frosty8104 — APPROVED** [review]

**Review by Valeriusdev — APPROVED** [review]
good work

File: src/components/MainSection.jsx (hunk 1 above)
Comment (NadiiaLashtun) [reply src/components/MainSection.jsx:24]: Thank you. Of course we can use it if necessary.

**Review by KseniiaRiabova — APPROVED** [review]
@NadiiaLashtun good job!

**Pull request merged**